py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -402,6 +413,7 @@ def index_dtype_validator(node: Node) -> bool:
     return True
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.index.Tensor)
```

Comment:
If this PR removes the `index_dtype_validator` constraint on the `index` operator, remove the below line to avoid duplicate registrations of the same operator.